fxa/oauth.py

```diff
@@ -239,12 +260,39 @@ def verify_token(self, token, scope=None):
             resp = None
 
         if resp is None:
-            url = '/verify'
-            body = {
-                'token': token
-            }
-            resp = self.apiclient.post(url, body)
-
+            # TODO we want to fetch
+            # https://oauth.accounts.firefox.com/.well-known/openid-configuration
+            # and then get the jwks_uri key to get the /jwks url, but we'll
+            # just hardcosde it like this for now until we are closer to
+            # production. We need to do this because oauth.accounts.firefox.com
+            # does not yet respond to .well-known, but accounts.firefox.com
+            # does; for consistency, oauth.accounts.firefox.com should respond
+            # to .well-known.
+
+            keys = self.apiclient.get('/jwks').get('keys', [])
+            resp = None
+            try:
+                for k in keys:
+                    try:
+                        resp = self._verify_jwt_token(json.dumps(k), token)
+                        break
+                    except jwt.exceptions.InvalidSignatureError:
+                        # It's only worth trying other keys in the event of
+                        # `InvalidSignature`; if it was invalid for other reasons
+                        # (e.g. it's expired) then using a different key won't
+                        # help. OTOH you might prefer to let the server do a bit of
+                        # extra work just to keep the control-flow simpler.
```

Comment:
nit: the "OTOH" sentence here is more of a comment to you in this PR rather than to our future selves, and probably doesn't make sense to include in the code as a comment here :-)